app/page.tsx

```diff
@@ -1,35 +1,140 @@
 'use client';
 
 import Link from 'next/link';
-
+import Image from 'next/image';
 import Container from '@/components/layout/Container';
-import { Button } from '@/components/ui/button';
+import Buttons from '@/components/Buttons';
 import useUser from '@/hooks/useUser';
+import { useRouter } from 'next/navigation';
 
 export default function LandingPage() {
-  const { user, memberships, isPending } = useUser();
+  const router = useRouter();
+  const { user } = useUser();
+
+  const handleButtonClick = () => {
+    if (user) {
+      router.push('/team/create');
+    } else {
+      router.push('/auth/login');
+    }
+  };
 
-  if (isPending && !user) {
-    return <div>사용자 정보를 불러오는 중입니다...</div>;
-  }
+  return (
+    <div className="w-full">
+      <div className="relative flex h-pr-1080 w-full justify-center bg-[url('/images/landing/img-Landing-bg.png')] bg-cover bg-center bg-no-repeat">
+        <div className="mt-pr-84 flex flex-col items-center justify-start gap-pr-20">
+          <div className="flex items-center justify-center gap-pr-20">
+            <div className="text-48sb">함께 만들어가는 투두 리스트</div>
+            <Image
+              src="/images/landing/img-Tool.svg"
+              alt="Tool"
+              width={56}
+              height={56}
+            />
+          </div>
 
```

Comment:
margin-top이 없어서 bg가 header에 묻히네요 headers height에 맞게 마진 주셔야 할 것 같습니다~
```suggestion
      <div className="relative mt-pr-60 flex h-pr-1080 w-full justify-center bg-[url('/images/landing/img-Landing-bg.png')] bg-cover bg-center bg-no-repeat">
```